lib/mhv_logging/configuration.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+module MHVLogging
+  # Configuration class used to setup the environment used by client
+  class Configuration < Rx::Configuration
```

Comment:
So the requests go to URLs like `#{host}/mhv-api/patient/v1/activity/auditlogout`? If that's true, then we still have a problem with breakers for this. It has to be able to distinguish requests to the different services in order to tell them apart. Take a look at the matcher in Rx::Configuration.